### DISCUSSION TITLE
Touchpoint OPT-301-Lot-EmployLegis  text reference seems wrong om all templates

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#759] [Q&A]
Touchpoint OPT-301-Lot-EmployLegis  text reference seems wrong om all templates.
It is defined as:
			{OPT-200-Organization-Company[(OPT-200-Organization-Company == OPT-301-Lot-EmployLegis) or (OPT-201-Organization-TouchPoint == OPT-301-Lot-EmployLegis)]} #{auxiliary|text|organisation-providing-info-environment} // Employment legislation information provider (LOT)

It uses the text reference organisation-providing-info-environment which is the text reference for OPT-301-Lot-EnvironLegis.  I assume that the text reference for OPT-301-Lot-EmployLegis should be another text reference ( i think it should be organisation-providing-info-employment

**Comment by pdonohoe** [comment]
Thank you @kletersteegp for pointing out this error. It will be fixed with the next release of the SDK, 1.10
